file-formats/uist/type/zif_aff_uist_v1.intf.abap

```diff
@@ -0,0 +1,56 @@
+INTERFACE zif_aff_uist_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Properties</p>
+    "! Properties
+    BEGIN OF ty_general_information,
+      "! <p class="shorttext">Title</p>
+      "! Space Template Title
+      "! $required
+      title         TYPE zif_aff_types_v1=>ty_description_100,
+      "! <p class="shorttext">Sort Priority</p>
+      "! Sort priority
+      "! $minimum: -999.999
+      "! $maximum: 999.999
+      sort_priority TYPE p LENGTH 7 DECIMALS 3,
+      "! <p class="shorttext">Base Space Template Name</p>
+      "! Base Space Template Name
+      base_space_template_name       TYPE c LENGTH 35,
+    END OF ty_general_information.
+
+  TYPES:
+    "! <p class="shorttext">SAP Fiori Page Template</p>
+    "! SAP Fiori Page Template
+    BEGIN OF ty_page,
+      "! <p class="shorttext">Name</p>
+      "! Page Template Name
+      name          TYPE c LENGTH 35,
+    END OF ty_page,
+
+    ty_pages TYPE TABLE OF ty_page WITH KEY name.
```

Comment:
only a question: Do you want to have the entries sorted? If you want to have it as a standard table it would be more recognizable to explicitly declare it as standard table 

```suggestion
    ty_pages TYPE STANDARD TABLE OF ty_page WITH KEY name.
```